cmd/crc/cmd/daemon.go

```diff
@@ -143,7 +143,12 @@ func run(configuration *types.Configuration) error {
 		mux := http.NewServeMux()
 		mux.Handle("/network/", http.StripPrefix("/network", vn.Mux()))
 		mux.Handle("/api/", http.StripPrefix("/api", api.NewMux(config, newMachine(), logging.Memory, segmentClient)))
-		if err := http.Serve(listener, handlers.LoggingHandler(os.Stderr, mux)); err != nil {
+		s := &http.Server{
+			Handler:      handlers.LoggingHandler(os.Stderr, mux),
+			ReadTimeout:  10 * time.Second,
+			WriteTimeout: 10 * time.Second,
+		}
+		if err := s.Serve(listener); err != nil {
```

Comment:
Fwiw, the commit fixing `make lint` could come before the commit updating golangci-lint to the latest version, this way `make check` would work for all commits in this PR. Not really important, but can be useful in projects using `git bisect` intensively.

Reply:
reordered the commits!